Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV PATH="./node_modules/.bin:$PATH:/usr/local/nginx/sbin/:/usr/local/luajit/bin
     DISPLAY=:99.0 \
     SKIP_ASSETS="1" \
     TZ=:/etc/localtime \
-    LD_LIBRARY_PATH=/opt/oracle/instantclient_12_2/ \
+    LD_LIBRARY_PATH=/opt/oracle/instantclient_12_2/:$LD_LIBRARY_PATH \
```

Comment:
This does look like a valid change, but I am wondering if it is related to the bundler version update?

Does this change fix some issue that comes up when using the latest bundler version?